pluginmage/errors.go

```diff
@@ -0,0 +1,21 @@
+package pluginmage
+
+import "fmt"
+
+const (
+	ErrorsURL = "https://github.com/mattermost/mattermost-plugin-legal-hold/blob/main/plugin_magefile/ERROR_CODES.md#"
+
+	ErrInitGnuTar = "init-gnu-tar"
+	ErrInitGo     = "init-go"
+	ErrInitNpm    = "init-npm"
+)
+
+// buildErrorURL builds the URL for the error code
+func buildErrorURL(code string) string {
+	return fmt.Sprintf("%s%s", ErrorsURL, code)
+}
```

Comment:
I'm surprised we need to do more than just inform the user to install the requisite tooling -- is there not enough context in the error message when this gets printed?